seqr/views/utils/test_utils.py

```diff
@@ -117,3 +117,204 @@ def _proxy_request_stub(*args, **kwargs):
     'constraints', 'diseaseDesc', 'functionDesc', 'notes', 'omimPhenotypes', 'mimNumber', 'mgiMarkerId', 'geneNames',
 }
 GENE_DETAIL_FIELDS.update(GENE_FIELDS)
```

Comment:
You also need to update the test you copied these from to import these

Reply:
I updated seqr/utils/elasticsearch/es_utils_tests.py.